src/solvers/dgsem_p4est/containers.jl

```diff
@@ -13,7 +13,7 @@ mutable struct P4estElementContainer{NDIMS, RealT <: Real, uEltype <: Real, NDIM
     # [jacobian_i, jacobian_j, node_i, node_j, node_k, element] where jacobian_i is the first index of the Jacobian matrix,...
     jacobian_matrix::Array{RealT, NDIMSP3}
     # Contravariant vectors, scaled by J, in Kopriva's blue book called Ja^i_n (i index, n dimension)
-    contravariant_vectors::Array{RealT, NDIMSP3}   # [dimension, index, node_i, node_j, node_k, element]
+    contravariant_vectors::PtrArray{RealT, NDIMSP3}   # [dimension, index, node_i, node_j, node_k, element]
```

Comment:
That's not a full type information so it will still cause allocations. You should better use a type parameter here.

Reply:
Done in https://github.com/amrueda/Trixi.jl/commit/500db958c8fad5777ac9e3e7e605510484a6b147